Storage.md

```diff
@@ -0,0 +1,304 @@
+```
+
+To clear the collection, but keep some items which will always be in the collection, utilize ItemsHoldr's `valueDefault` property.
+
+```typescript
+let ItemsHolder: ItemsHoldr = new ItemsHoldr({
+    values: {
+        color: {
+            valueDefault: "red"
+        }
+    }
+});
+ItemsHolder.clear(); // "color" is still in the collection with a reset value of "red"
+```
+
+## Elements
+
+### *Note:* This part of ItemsHoldr will be placed into another module in the future.
+
+HTML elements stored within ItemsHoldr will have their values on the page updated to their changed value.
+
+```typescript
+ItemsHolder.addItem("color", {
+    hasElement: true,
+    element: {}
+});
+ItemsHolder.setItem("color", "red");
+```
+
+To signal if a container should be made to hold HTML elements, assign a value to `doMakeContainer`.
+
+```typescript
+let ItemsHolder: ItemsHoldr = new ItemsHoldr({ doMakeContainer: true });
+```
+
+With this, any HTML items passed in at the time of construction are held in ItemsHolder's `container` object.
+Each of the item's elements are added to the container as its children.
+It can be used to display a number of elements grouped together.
+
+```typescript
+let ItemsHolder: ItemsHoldr = new ItemsHoldr({
+    doMakeContainer: true,
+    values: {
+        color: {
+            valueDefault: "red",
+            hasElement: true,
+            element: {}
+        }
+    }
+});
+
+// color is in the container, but speed is not
+ItemsHolder.addItem("speed", {
+    valueDefault: 10,
+    hasElement: true,
+    element: {}
+});
+```
+
+### Display Changes
+
+If you want certain values to be represented differently on the page, make use of `displayChanges`.
+These changes are hardcoded values that replace specified values when element items are updated.
+
+```typescript
+let ItemsHolder = new ItemsHoldr({
+    displayChanges: {
+        Infinity: "INF";
+    }
+});
+
+ItemsHolder.addItem("limit", {
+    value: "4000",
+    hasElement: true,
+    element: {}
+});
+ItemsHolder.setItem("limit", "Infinity");
+```
+
+`hasDisplayChange` checks to see if a value has a recorded change.
+
+```typescript
+ItemsHolder.hasDisplayChange("Infinity");
+```
+
+...and `getdisplayChange` returns the entry to replace the value with.
+
+```typescript
+let newValue: string = ItemsHolder.getDisplayChange("Infinity");
+```
+
+### Container Visibility
+
+`hideContainer` hides the container from view.
+
+```typescript
+ItemsHolder.hideContainer();
+```
+
+...while `displayContainer` makes the container visible.
+
+```typescript
+ItemsHolder.displayContainer();
+```
+
+More can be read about ItemsHoldr on its [Readme](https://github.com/FullScreenShenanigans/ItemsHoldr/blob/master/README.md).
+
+# StateHoldr
+
+StateHoldr is a module for tracking changes of items in ItemsHolder in objects called collections.
+A collection is an object whose keys are item names in ItemsHolder and whose values are key-value pairs of item attributes and values.
```

Comment:
That was me trying to explain changes were keyed by item names.
How about this to replace the rest of that section.

Changes are key-value pairs of attributes for items in ItemsHolder and values.
A collection is a group of these changes keyed by the item name.
Collections allow for various attributes for a single item to be grouped together versus storing individual items.

To be honest I'm not sure how to best go about explaining collections and changes